src/LanguageHandler.js

```diff
@@ -1,5 +1,49 @@
 import { toIterablePromise } from './promiseUtils';
 
+function _isWildCard(tag) {
+  return tag === '*';
+}
+
+function _matchLangTag(left, right) {
+  const matchInitial = new RegExp(`/${left}/`, 'iu');
+  return matchInitial.test(`/${right}/`);
+}
+
+// TODO: Not an XPath function
+// TODO: Publish as package
+// https://www.ietf.org/rfc/rfc4647.txt
+// https://www.w3.org/TR/sparql11-query/#func-langMatches
+export function langMatches(tag, range) {
```

Comment:
It appears that with this addition you are now matching against things like `en-gb`; I think it would be useful to have tests with these kinds of patterns in the `execute-query-multilinigual-test.js`; testing both the SPARQL query produced (`.sparql`) and the execution results.

In these cases I *think* the filter on the executed sparql query (prior to your filtering in the `_handle` function) should be  `FILTER("en")` whilst I suspect you are doing `FILTER("en-gb")`; so could you please add some tests around this.